pom.xml

```diff
@@ -1575,6 +1599,15 @@
                 </pluginManagement>
             </build>
         </profile>
+        <profile>
+            <id>jdk11plus</id>
+            <activation>
+                <jdk>[11,)</jdk>
+            </activation>
+            <properties>
+                <surefire.argLine>-XX:+IgnoreUnrecognizedVMOptions -Xshare:off</surefire.argLine>
+            </properties>
+        </profile>
```

Comment:
Small comment about why this exists would be appreciated.

Reply:
done